tests/eru.Application.Tests/FakeDbContext.cs

```diff
@@ -27,9 +27,14 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             {
                 x.HasKey(y => y.Id);
                 x.Property(y => y.Id)
-                    .HasMaxLength(255);
+                    .HasMaxLength(255)
+                    .ValueGeneratedOnAdd();
```

Comment:
Why is it here?

Reply:
Since [EF Core 3.0](https://stackoverflow.com/questions/59134406/unable-to-track-an-entity-of-type-because-primary-key-property-id-is-null) string primary keys are not automatically fulfiled with GUIDs when the record is created. PropertyBuilder.ValueGeneratedOnAdd method changes this behavior.

Reply:
ok